internal/hook/item_hook_handler.go

```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 	"time"
 
-	uuid "github.com/gofrs/uuid"
+	uuid "github.com/google/uuid"
```

Comment:
not quite sure why we need to rename this.
why not just make it 
"github.com/google/uuid" 
in the import chunk?

Reply:
They are different library, or two different provider of uuid. I am not sure why `gofrs/uuid` was used either, but as far as my understanding, it is safe to replace it with `google/uuid`

Reply:
Not sure the history, but IMO, we can rename it

Reply:
Done